nc_news_model.js

```diff
@@ -0,0 +1,17 @@
+const db = require("./db/connection")
+
+function selectAllTopics(){
+    return db
+    .query("SELECT slug, description FROM topics;")
+    .then((result)=>{
+        console.log(result.rows, "result")
+        return result.rows
+    }).catch((err)=>{
+    })
```

Comment:
What happens when we catch an err? We still want to move it along the middleware, so you should still invoke next with the err :)